src/pages/projects/ProjectGeneralInfo.tsx

```diff
@@ -27,13 +28,22 @@ export const ProjectGeneralInfo = ({ project }: ProjectGeneralInfoProps) => {
 
   return (
     <StyledGeneralInfo data-testid={dataTestId.projectLandingPage.generalInfoBox}>
-      <div>
-        <Typography variant="overline">{t('project.project_id')}</Typography>
-        <Typography>{getValueByKey('CristinIdentifier', project.identifiers)}</Typography>
-        <Typography variant="overline">{t('project.coordinating_institution')}</Typography>
-        <Typography>{getProjectCoordinatingInstitutionName(project) ?? '-'}</Typography>
-        <Typography variant="overline">{t('project.project_manager')}</Typography>
-        <Typography>
+      <Typography component="h2" sx={visuallyHidden}>
+        {t('project.about_project')}
+      </Typography>
```

Comment:
Trenger kanskje ikke denne?
Virker fornuftig å bare legge inn skjulte headings der man må ha det for at det skal gi mening for skjermlesere, men her ser det ut som det virker fint uten både for skjermleder og visuelt

Reply:
Ja, var litt usikker på denne. Vi har en visuell header på LP for resultat, så tanken var å speile den layouten for skjermlesere.